packages/yew-bootstrap/src/component/navbar.rs

```diff
@@ -33,7 +43,16 @@ impl Component for NavDropdownItem {
 
         html! {
             <li>
-                <a class="dropdown-item" href={props.url.clone()}>{props.text.clone()}</a>
+                <a
+                    class="dropdown-item"
+                    href={props.url.clone()}
+                    onclick={props.onclick.clone()}
+                >
+                    if let Some(icon) = props.icon {
+                        {icon}{" "}
```

Comment:
What's with the quoted space character?

Reply:
I'd done that to provide a little bit of extra space between the icon an the text.

Without the space, it looks like this:

![Screenshot 2024-09-08 at 10 49 59](https://github.com/user-attachments/assets/68ae86f0-9af5-42da-829c-a2cf65308fb5)

I've also tried using the [icon link helper](https://getbootstrap.com/docs/5.3/helpers/icon-link/), but that looks the same.

I tried switching `build::bootstrap_icons()` to use the same `<svg class=...><use ...>` syntax as the examples on that page:

```rust
sorted_icons.insert(lower_kebab_case, format!(
    "    pub const {upper_snake_case}: BI = BI(\"<svg class=\\\"bi\\\" aria-hidden=\\\"true\\\"><use xlink:href=\\\"#{lower_kebab_case}\\\"></use></svg>\");\n"
));
```

...but it looks like that [Bootstrap Icons' SVG sprite sheet](https://icons.getbootstrap.com/#usage) needs either embedding or re-hosting on the same domain to work correctly... so that's entirely broken:

![Screenshot 2024-09-08 at 11 04 42](https://github.com/user-attachments/assets/aff3b2a7-06c2-4f78-b3e7-bd8c717876ce)